lib/dotcom_web/components/system_status/subway_status.ex

```diff
@@ -141,18 +141,45 @@ defmodule DotcomWeb.Components.SystemStatus.SubwayStatus do
     |> Enum.flat_map(&rows_for_route(&1, include_alert: true))
   end
 
-  defp maybe_collapse_rows(rows) when length(rows) > @max_rows, do: collapse_rows(rows)
-  defp maybe_collapse_rows(rows), do: rows
+  defp collapse_rows_if_needed(rows) do
```

Comment:
suggestion(non-blocking): I like this! But you probably don't need this wrapping function and can put the three lines in there directly :D

Reply:
I'm on the fence, but I think I prefer it as-is. As it currently is, it keeps the "should we think about collapsing" separate from the "what does collapsing mean", which makes `status_to_rows/1` read more cleanly (to me, at least 😇)